.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+sokiable*
+.idea
```

Comment:
Ensure line ending at the end of file. If you using IntelliJ IDE check setting "Ensure line feed on Save" under Editor -> General